server/aws-lsp-codewhisperer/src/language-server/agenticChat/context/agenticChatTriggerContext.ts

```diff
@@ -242,7 +242,7 @@ export class AgenticChatTriggerContext {
         if (textDocumentIdentifier?.uri === undefined) {
             return
         }
-        const textDocument = await this.getTextDocument(textDocumentIdentifier.uri)
+        const textDocument = await this.getTextDocumentFromUri(textDocumentIdentifier.uri)
```

Comment:
can we load from fs here as well to keep it consistent?

Reply:
I'm not familiar with most of this code flow. Do you know what the impact would be if we avoid using the workspace altogether?